### PR TITLE
Replace String.format calls with simple string concats or parametered log call

### PR DESCRIPTION
**Pull request by Janmm14**
Not really much left of the previous PR #3141 (which is good, the code was better than anticipated), but:

This does not clutter code and makes it a little faster at the same time.

### CODE REVIEW

File: proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java [hunk 1]
```diff
@@ -83,4 +89,13 @@ public void run()
 
         cancel();
     }
+
+    private static LogRecord log(Throwable t, String msg, Object[] parameters, Logger logger)
```
Comment (md-5) [inline proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java:93]: What's the purpose of this change?

Comment (Janmm14) [reply proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java:93]: Have parameters and an exception in a logrecord. For some reason its missinng in the log overloads

Comment (md-5) [reply proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java:93]: Just concat it, or yet leave it - how many times a second do you realistically expect a plugin to throw an exception in a task?

Comment (Janmm14) [reply proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java:93]: I don't see the problem in adding this method. Code is clear.

Comment (md-5) [reply proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java:93]: It's pointless code though.
If you were writing this code from scratch it's highly unlikely you'd make a method like that

Comment (Janmm14) [reply proxy/src/main/java/net/md_5/bungee/scheduler/BungeeTask.java:93]: In that case, edit it yourself please how you would like to have it (I have allow edits from maintainers on).
